libs/@guardian/source-react-components/scripts/create-icons/create-icon-component.ts

```diff
@@ -120,7 +120,11 @@ export const createIconComponent = async ({
 	const label = labels[icon.name];
 
 	if (!label) {
-		throw new Error(`No label found for ${icon.name}`);
+		// This error is thrown when the accessible label data for the icon is missing in Figma.
+		// If you need to generate the icon components quickly and fix the label later, you can comment out this line.
```

Comment:
I think we might be wiser not suggesting this lest we set a precedent for bad habits - the comment on the line above tells people what they need to do.

Reply:
agreed - will remove